From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Felix <29801805+ImGxrke@users.noreply.github.com>
Date: Sun, 17 Mar 2024 17:49:34 +0100
Subject: [PATCH] Workaround postion desync


diff --git a/src/main/java/net/minecraft/server/level/ServerEntity.java b/src/main/java/net/minecraft/server/level/ServerEntity.java
index a2279262c93408c11f5d2290b48fd794975e8cfe..0cf1e763dd8857ade87ae8c8bb631516d6efc11f 100644
--- a/src/main/java/net/minecraft/server/level/ServerEntity.java
+++ b/src/main/java/net/minecraft/server/level/ServerEntity.java
@@ -382,6 +382,7 @@ public class ServerEntity {
             }
         }
 
+        sender.accept(new ClientboundTeleportEntityPacket(this.entity));
     }
 
     private void sendDirtyEntityData() {
